go.sum

```diff
@@ -150,7 +150,6 @@ github.com/stretchr/testify v1.4.0/go.mod h1:j7eGeouHqKxXV5pUuKE4zz7dFj8WfuZ+81P
 github.com/stretchr/testify v1.6.1/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/h/Wwjteg=
 github.com/stretchr/testify v1.7.1/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/h/Wwjteg=
 github.com/stretchr/testify v1.8.0 h1:pSgiaMZlXftHpm5L7V1+rVB+AZJydKsMxsQBIJw4PKk=
-github.com/stretchr/testify v1.8.0/go.mod h1:yNjHg4UonilssWZ8iaSj1OCr/vHnekPRkoO+kdMU+MU=
```

Comment:
It looks like we should remove the other line above for `github.com/stretchr/testify v1.8.0`?

- What was the reason behind downgrading this version to v1.7.1?
- Should we have entries for the other require you added `github.com/stretchr/objx v0.4.0`?

Reply:
So I didn't downgrade anything manually. This would have been generated when I ran one of the go commands.  Can you a have a look at the whole file, if you haven't already? It seems like a different version are referenced throughout this file.